util/min_test.go

```diff
@@ -0,0 +1,27 @@
+package util
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMin(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
+		x int
+		y int
+		expected int
+	}{
+		{1, 2, 1},
+		{100, 4, 4},
+		{0, 25, 0},
+		{-1, 1, -1},
+		{0, -1, -1},
```

Comment:
Should we handle the corner cases mentioned here? https://golang.org/pkg/math/#Min. Also, check two equal numbers.

Reply:
Good catch on the 2 equal number check - ty! Added and feels like that made the test better :)

re: corner cases - I looked over those just now.. it seems most of them are only issues when working with `float64` types... I can't think of a case we'd get something like `-Inf` or `-0` since we are only going to be working with `int`s.